sys/netbsd/fs.txt

```diff
@@ -45,20 +45,20 @@ iovec_out {
 }
 
 stat {
-	dev	int16
-	ino	int32
+	dev	int64
+	ino	int64
 	mode	int16
-	nlink	int16
-	rdev	int16
+	nlink	int32
+	rdev	int64
 	size	int32
```

Comment:
@zoulasc I couldn't find the type of `off_t` and `mode_t`, so I left them as they are. Thanks for pointing out.